configuration/linux-capability.md

```diff
@@ -0,0 +1,181 @@
+# Linux capability
+
+This article shows configuration and dependent gem installation instruction for enabling Linux capability module on Fluentd core.
+
+## Prerequisites
+
+* gcc and make etc. for building C extension sources
+* libcap-ng package and its depelopment package
+  * libcap-ng-dev on Debian GNU/Linux and Ubuntu
+  * libcap-ng-devel on CentOS 7/8, Fedora 33, AmazonLinux 2
+* pkg-config package for linking libcap-ng library
+* Ruby and its development packages
+  * ruby-dev on Debian GNU/Linux and Ubuntu
+  * ruby-devel on CentOS 7/8, Fedora 33, AmazonLinux 2
+* `setcap` command
+  * libcap2-bin on Debian GNU/Linux and Ubuntu
+  * libcap on CentOS 7/8, Fedora 33, AmazonLinux 2
+* `filecap` command
+  * libcap-ng-utils on Debian GNU/Linux and Ubuntu
+  * libcap-ng-utils on CentOS 7/8, Fedora 33, AmazonLinux 2
+
```

Comment:
Should mention `Fluentd v1.12 or later`.

Reply:
Oops. I'll add it.

Reply:
Fixed https://github.com/fluent/fluentd-docs-gitbook/pull/248/commits/16f9e0e93bb60b54b352769656a1524177bb215b